src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -189,7 +189,7 @@ public String t3XMP() {
                 + "</rdf:Seq></bibtex:editor>" + "<bibtex:bibtexkey>Clarkson06</bibtex:bibtexkey>"
                 + "<bibtex:journal>International Journal of High Fidelity</bibtex:journal>"
                 + "<bibtex:booktitle>Catch-22</bibtex:booktitle>" + "<bibtex:pdf>YeKis03 - Towards.pdf</bibtex:pdf>"
-                + "<bibtex:keywords>peanut,butter,jelly</bibtex:keywords>"
+                + "<bibtex:keywords>peanut, butter, jelly</bibtex:keywords>"
```

Comment:
To make the tests pass... But I might not have analysed that carefully exactly where it should be changed, but changed all keywords to contain ', ' as separator. You are probably correct that here it is better not to change it.